docs/developer/testing.md

```diff
@@ -0,0 +1,188 @@
+# Testing guidance
+
+Prior to reviewing with key stakeholders, it's recommended to perform this series of tests to build the confidence that you've met your customers' expectations.
+
+## Quality checklist
+
+1. Test each page for design and content issues
+    1. Test screen sizes from viewport 1 to 6
```

Comment:
I'm not sure this is relevant in our current architecture. This is confusing as we aren't building "pages" or dealing with a fixed number of viewports. What package is this for? Is this generic documentation?

Reply:
Upon reviewing the quality checklist, I'm not sure I understand how it fits within this documentation and FAST

Reply:
We're providing guidance from lessons we learned building Web sites and Apps.  I believe this guidance help add credibility and confidence in those considering using FAST-DNA.

Reply:
We can always expand from here to make it more granular. This is the same process we used on MWF and FluentWeb.